libs/sdk-model/src/dashboard/alert.ts

```diff
@@ -1,12 +1,12 @@
 // (C) 2019-2022 GoodData Corporation
-import { ObjRef, isObjRef } from "@gooddata/sdk-model";
-import { IFilterContext, IFilterContextDefinition, isFilterContextDefinition } from "./filterContext";
-import { IDashboardObjectIdentity } from "./common";
 import isEmpty from "lodash/isEmpty";
+import { ObjRef, isObjRef } from "../objRef";
+import { IDashboardObjectIdentity } from "./common";
+import { IFilterContext, IFilterContextDefinition, isFilterContextDefinition } from "./filterContext";
 
 /**
  * Common widget alert properties
- * @public
+ * @alpha
```

Comment:
Several more changes from public to alpha. Is this wanted?

Reply:
Another revert: https://github.com/gooddata/gooddata-ui-sdk/commit/20469cf8480ad6439eb3942be7e8e8e72b3e1004#diff-969dbfba353c20c17f7c9bbd6e1b88723bd89bfb768a7b1199e02fe08e47a0cbL9